packages/flutter/lib/src/material/calendar_date_picker.dart

```diff
@@ -1074,27 +1082,40 @@ class _DayPickerGridDelegate extends SliverGridDelegate {
 
 const _DayPickerGridDelegate _dayPickerGridDelegate = _DayPickerGridDelegate();
 
-/// A scrollable list of years to allow picking a year.
+/// A scrollable grid of years to allow picking a year.
+///
+/// The year picker widget is rarely used directly. Instead, consider using
```

Comment:
Since it's a private class, it can't be used directly, so perhaps this sentence should be removed

Reply:
I will be exposing it as a public class to restore some functionality that we deprecated, so I will leave this as is so we don't have to change it again.